modules/assim.batch/R/pda.emulator.R

```diff
@@ -281,6 +272,8 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
           SS.list[[inputi]] <- cbind(X, error.statistics[[inputi]])
       } # if-block
         
+      # remove failed runs
+      SS.list[[inputi]] <- SS.list[[inputi]][!rowSums(is.na(SS.list[[inputi]])), ]
     } # for-loop
```

Comment:
for now I'm completely removing all the parameter sets of the failed runs because I don't know why the model failed, it may not only be due to unlikeliness of parameter combinations, so I'm not assuming anything and removing them.

But for how many failed runs should we allow this @mdietze?